src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="imac">
+        <img
+          src="images/imac.jpeg"
+          alt="imac photo"
+        />
+      </div>
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="star">
+        <div class="star__star">
+          <div class="star__stars-active"></div>
+          <div class="star__stars-active"></div>
+          <div class="star__stars-active"></div>
+          <div class="star__stars-active"></div>
+          <div class="star__stars"></div>
+        </div>
```

Comment:
This doesn't meet the requirment - all stars--N modifiers work as expected (Highlight first N stars). You should decide whether which stars are active with this one modifier class `stars--4`